src/analysis/signals.rs

```diff
@@ -10,12 +10,14 @@ use version::Version;
 
 #[derive(Debug)]
 pub struct Info {
+    pub object_name: String,
```

Comment:
This is only added so that we can print it in codegen later for action signals :)